.github/ISSUE_TEMPLATE/nightly_build_failure.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Nightly build failure report
+about: Lightweight incident report for nightly build failure
+title: Nightly Build Failure XXXX-XX-XX
+labels:
+  - bug
+  - nightly-builds
+assignees: ''
+---
+
+# Overview
+
+What do you think happened, if anything?
+
+What next steps do we need to do?
+
+```[tasklist]
+# Next steps
+- [ ] verify that [S3 distribution bucket](https://s3.console.aws.amazon.com/s3/buckets/pudl.catalyst.coop?region=us-west-2&bucketType=general&prefix=nightly/&showversions=false) was updated at the expected time
```

Comment:
These steps seem more oriented towards "make sure the updates failed as expected / clean up after failure" and not "identify the source of the failure and fix it", which is the intended goal of this issue template. I'm a bit confused about why we'd want to check them here, rather than make an issue to address any existing issues with the nightly build failures working as expected, if that makes sense?

Reply:
My thought was basically that we should *at least* always check this, and then any actual investigative tasks should be added by the creator.